source/pixform.h

```diff
@@ -173,6 +100,8 @@
 #define MAX_RED            MAX_RED_D(PIXEL_FORMAT)
 #define MAX_BLUE           MAX_BLUE_D(PIXEL_FORMAT)
 #define MAX_GREEN          MAX_GREEN_D(PIXEL_FORMAT)
+#define RED_SHIFT_BITS     RED_SHIFT_BITS_D(PIXEL_FORMAT)
+#define GREEN_SHIFT_BITS   GREEN_SHIFT_BITS_D(PIXEL_FORMAT)
```

Comment:
I think you also need to add a guard for PSP here, or otherwise it will break compilation.